clients/src/main/resources/common/message/DescribeProducersResponse.json

```diff
@@ -35,7 +35,7 @@
         { "name": "ErrorMessage", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
           "about": "The partition error message, which may be null if no additional details are available" },
         { "name": "ActiveProducers", "type": "[]ProducerState", "versions": "0+", "fields": [
-          { "name": "ProducerId", "type": "int64", "versions": "0+" },
+          { "name": "ProducerId", "type": "int64", "versions": "0+", "entityType": "producerId" },
           { "name": "ProducerEpoch", "type": "int32", "versions": "0+" },
```

Comment:
The type confuses me. `DescribeProducersResponse` and this protocol use `int64` but other protocols choose `int32`. Also, the type of `TransactionMetadata#producerEpoch` is `short` rather than `integer`. Which one is correct?

Reply:
The type should be `int16`. That is what we use in other APIs (e.g. `AddPartitionsToTxn` and `EndTxn`). It's been a while since I wrote this, so I'm not sure what I was thinking. Perhaps I was anticipating expansion of this field. It is probably better to be consistent since we can always bump the protocol in the future if needed. I will fix it here and consider a separate patch for `DescribeProducers`.